src/towncrier/test/test_check.py

```diff
@@ -476,14 +476,28 @@ def test_ignored_files(self, runner):
         """
         When `ignore` is set in config, files with those names are ignored.
         """
-        create_project("pyproject.toml", extra_config='ignore = ["template.jinja"]')
+        create_project(
+            "pyproject.toml",
+            extra_config='ignore = ["template.jinja", "star_wildcard*", "question_wildcard_?", '
+            '"seq_wildcard_[ab]"]',
+        )
 
         write(
             "foo/newsfragments/124.feature",
             "This fragment has valid name (control case)",
         )
         write("foo/newsfragments/template.jinja", "This is manually ignored")
         write("foo/newsfragments/.gitignore", "gitignore is automatically ignored")
+        write("foo/newsfragments/star_wildcard_foo", "Manually ignored with * wildcard")
+        write("foo/newsfragments/STAR_WILDCARD_bar", "Manually ignored with * wildcard")
+        write(
+            "foo/newsfragments/question_wildcard_1", "Manually ignored with ? wildcard"
+        )
+        write(
+            "foo/newsfragments/QUESTION_WILDCARD_1", "Manually ignored with ? wildcard"
+        )
+        write("foo/newsfragments/seq_wildcard_a", "Manually ignored with [] wildcard")
+        write("foo/newsfragments/SEQ_WILDCARD_b", "Manually ignored with [] wildcard")
```

Comment:
I think that we can keep it simple and have a single test example... otherwise, this test becomes too long and hard to follow.

If you want to check all possible values supported by fmatch, is best to write a separate tests... but I think that a single update is good enough.

```suggestion
```